src/items/equipment/tool_kit_xhinti_holdings_linguist’s_kit.json

```diff
@@ -0,0 +1,91 @@
+{
+  "_id": "bA7zT96w055N8EoA",
+  "name": "Tool Kit, Xhinti Holdings, Linguist’s Kit",
+  "type": "technological",
+  "flags": {
+    "core": {
+      "sourceId": "Compendium.sfrpg.equipment.u29xKStsNlJzi5I7"
+    }
+  },
+  "img": "systems/sfrpg/icons/equipment/technological%20items/tool-kit-xhinti-holdings.webp",
+  "system": {
+    "type": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "ammunitionType": "",
+    "attributes": {
+      "ac": {
+        "value": ""
+      },
+      "customBuilt": false,
+      "dex": {
+        "mod": ""
+      },
+      "hardness": {
+        "value": ""
+      },
+      "hp": {
+        "max": "",
+        "value": 6
+      },
+      "size": "medium",
+      "sturdy": false
+    },
+    "attuned": false,
+    "bulk": "L",
+    "capacity": {
+      "max": 0,
+      "value": 0
+    },
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": true,
+      "isOpen": true,
+      "storage": []
+    },
+    "critical": {
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>A tool kit is a set of specialized tools and devices not worth purchasing individually, but which as a collection are required for a given task or provide a circumstance bonus to certain skill checks.</p>\n<p>Xhinti Holdings, which has corporate offices on Salomo Station and factories on @UUID[Compendium.sfrpg.setting.gzXT0xwsj8kQAoj0]{Akiton}, produces a line of specialty tool kits that supplement those presented on page 221 of the Starfinder Core Rulebook. Xhinti tool kits include the following: a climbing kit (grants a +4 bonus to Athletics checks when climbing), demolitionist’s kit (grants a +4 bonus to Engineering checks when arming and disarming explosives), <strong>linguist’s kit</strong> (grants a +4 bonus to Culture checks when deciphering writing), portable weather station (grants a +4 bonus to Survival checks when predicting the weather), starship repair kit (grants a +4 bonus to Engineering checks when repairing the hull of a starship), and survivalist’s kit (grants a +4 bonus to Survival checks when enduring severe weather or living off the land). When using a demolitionist’s kit or starship repair kit for its specialized purpose, you don’t take the –2 penalty to Engineering checks if you don’t have an engineering kit.</p>"
```

Comment:
Is this Akiton link valid? It should link to a journal entry page. If it works, no worries

Reply:
I didn't touch that part.  If there is a link to Akiton, it existed before my suggested edits